src/issue/screens/edit-issue-comment.screen.js

```diff
@@ -0,0 +1,146 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { ScrollView, StyleSheet, TextInput, View } from 'react-native';
+import { ListItem } from 'react-native-elements';
+
+import { ViewContainer, SectionList, LoadingModal } from 'components';
+import { translate } from 'utils';
+import { colors, fonts, normalize } from 'config';
+import { editIssueComment } from '../issue.action';
+
+const styles = StyleSheet.create({
+  textInput: {
+    fontSize: normalize(12),
+    marginHorizontal: 15,
+    flex: 1,
+    color: colors.black,
+    ...fonts.fontPrimaryLight,
+  },
+  submitTitle: {
+    color: colors.green,
+    ...fonts.fontPrimary,
+  },
+  listItemContainer: {
+    flex: 1,
+  },
+  titleSmall: {
+    color: colors.primaryDark,
+    ...fonts.fontPrimarySemiBold,
+    fontSize: normalize(10),
+  },
+});
+
+const mapStateToProps = state => ({
+  language: state.auth.language,
+  repository: state.repository.repository,
+  isEditingComment: state.issue.isEditingComment,
+});
+
+const mapDispatchToProps = dispatch => ({
+  editIssueCommentByDispatch: (owner, repoName, issueTitle, issueComment) =>
+    dispatch(editIssueComment(owner, repoName, issueTitle, issueComment)),
+});
+
+class EditIssueComment extends Component {
+  props: {
+    editIssueCommentByDispatch: Function,
+    language: string,
+    repository: Object,
+    navigation: Object,
+    isEditingComment: boolean,
+    issue: Object,
+  };
+
+  state: {
+    issueComment: string,
+    issueCommentHeight: number,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      issueComment: this.props.navigation.state.params.comment.body,
+      issueCommentHeight: 0,
+    };
+  }
+
+  editIssueComment = () => {
+    const { navigation, editIssueCommentByDispatch } = this.props;
+    const { repository, comment } = this.props.navigation.state.params;
+
+    const repoName = repository.name;
+    const owner = repository.owner.login;
+    const text = this.state.issueComment;
+
+    editIssueCommentByDispatch(comment.id, owner, repoName, text).then(() =>
+      navigation.goBack()
+    );
+  };
+
+  render() {
+    const { language, repository, isEditingComment } = this.props;
+    const { issueComment } = this.state;
+    const issueTitle = this.props.navigation.state.params.issue.title;
+
+    return (
+      <ViewContainer>
+        {isEditingComment && <LoadingModal />}
+        <ScrollView>
+          {repository.full_name && (
+            <ListItem
```

Comment:
Feels kinda out of place to have this `ListItem` here, I think we can remove this to be honest (the user can easily see the back navigation step with the issue number)